### ISSUE TITLE
Nothing gets rendered until the program receives it's first message

### ISSUE THREAD

**Issue by szabba** [elm-community/elm-webgl#23]
As we've discussed on Slack with @w0rm , it seems that elm-webgl doesn't render anything until after the program receives it's first message. I've made a [gist](https://gist.github.com/szabba/13a90267523f122d259a16fe9c5954be) that exhibits this behaviour. The conditions in lines https://gist.github.com/szabba/13a90267523f122d259a16fe9c5954be#file-main-elm-L18-L23 can be used to make it work immediately or after a five second delay.

To poke at the code locally, you can clone the gist

``` bash
$ git clone https://gist.github.com/13a90267523f122d259a16fe9c5954be.git
```

**Comment by szabba** [comment]
:tada:

**Comment by w0rm** [comment]
@szabba I published 3.0.1 that contains this fix
